pdc/apps/common/viewsets.py

```diff
@@ -186,12 +191,18 @@ def initial(self, request, *args, **kwargs):
         if (request.method.lower() not in self.http_method_names or
                 not hasattr(self, request.method.lower())):
             return
-
         allowed_keys = drf_introspection.get_allowed_query_params(self)
         extra_keys = set(request.query_params.keys()) - allowed_keys
         if extra_keys:
             raise FieldError('Unknown query params: %s.' % ', '.join(sorted(extra_keys)))
 
+        # If 'ordering' in query parameter, check the key whether in fields.
+        if 'ordering' in request.query_params.keys():
+            if request.query_params.get('ordering') not in self.__class__.serializer_class.Meta.fields:
```

Comment:
Based on the documentation of `OrderingFilter`, it looks like the value can be a comma separated list of fields. That would not work with this check. Also, the field names can be prefixed with `-` to indicate reverse order, which would not work here either.